configs/config-ccm-psk-dtls1_2.h

```diff
@@ -23,19 +23,20 @@
  * See README.txt for usage instructions.
  */
 
```

Comment:
We need to check whether we're still executing the relevant tests in each configuration. The [logs](https://mbedtls.trustedfirmware.org/job/mbed-tls-pr-head/job/PR-9057-head/4/execution/node/5650/log/?consoleFull) show that we aren't, e.g.

```
 running ssl-opt.sh -f ECJPAKE.*nolog (config-thread.h)
 ECJPAKE: working, DTLS, nolog .......................................... SKIP
```

but comparing with [yesterday's nightly](https://mbedtls.trustedfirmware.org/job/mbed-tls-nightly-tests/579/execution/node/5455/log/?consoleFull) shows that this is at least partly a preexisting problem. We may want to put this pull request on hold until https://github.com/Mbed-TLS/mbedtls/issues/9063 is resolved.